src/pages/ShoppingList.js

```diff
@@ -91,61 +103,82 @@ const ShoppingList = ({ token }) => {
 
         if (!(item.lastPurchaseDate == null)) {
             let lastEstimate;
             item.nextPurchaseDate
-              ? (lastEstimate = item.nextPurchaseDate)
-              : (lastEstimate = item.timeFrame);
+                ? (lastEstimate = item.nextPurchaseDate)
+                : (lastEstimate = item.timeFrame);
             let lastPurchaseDate = item.lastPurchaseDate;
-            let today = moment(Date.now())
+            let today = moment(Date.now());
             let lastPurchase = moment(lastPurchaseDate);
             let latestInterval = today.diff(lastPurchase, 'days');
 
             let db = fb.firestore();
             let nextPurchaseDate = calculateEstimate(
-              item.lastEstimate,
-              latestInterval,
-              item.numOfPurchases
+                item.lastEstimate,
+                latestInterval,
+                item.numOfPurchases
             );
             db.collection(userToken)
-              .doc(e.target.name)
-              .update({
-                lastPurchaseDate,
-                numOfPurchases: numberOfPurchases,
-                latestInterval,
-                lastEstimate,
-                nextPurchaseDate,
-                isChecked: e.target.checked
-              })
-              .then(function() {
-                getShoppingList();
-            });
-          } else {
+                .doc(e.target.name)
+                .update({
+                    lastPurchaseDate,
+                    numOfPurchases: numberOfPurchases,
+                    latestInterval,
+                    lastEstimate,
+                    nextPurchaseDate,
+                    isChecked: e.target.checked,
+                })
+                .then(function() {
+                    getShoppingList();
+                });
+        } else {
             let lastPurchaseDate = moment(Date.now()).format();
             let db = fb.firestore();
             db.collection(userToken)
-              .doc(e.target.name)
-              .update({
-                  isChecked: e.target.checked,
-                  lastPurchaseDate,
-                  numOfPurchases: numberOfPurchases
+                .doc(e.target.name)
+                .update({
+                    isChecked: e.target.checked,
+                    lastPurchaseDate,
+                    numOfPurchases: numberOfPurchases,
                 })
-              .then(function() {
+                .then(function() {
                     getShoppingList();
-              });
+                });
         }
     };
 
+    const deleteItem = item => {
+        let db = fb.firestore();
+        db.collection(userToken)
+            .doc(item.id)
+            .delete()
+            .then(() => (getShoppingList(), setModal(false)));
+    };
+
     const filteredList = shoppingListItems.filter(item => {
         return item.itemName.toLowerCase().includes(filterString.toLowerCase());
     });
-
+    console.log('Modal status: ', modal);
```

Comment:
I think we can remove this if we're done testing 😄

Reply:
Done.  Just pushed the update!